sdk/anomalydetector/ai-anomaly-detector/README.md

```diff
@@ -41,15 +45,15 @@ npm install @azure/ai-anomaly-detector
 
 To create a client object to access the Anomaly Detector API, you will need the `endpoint` of your Anomaly Detector resource and a `credential`. The Anomaly Detector client can use either Azure Active Directory credentials or an API key credential to authenticate.
 
-You can find the endpoint for your Anomaly Detector resource either in the [Azure Portal][azure_portal] or by using the [Azure CLI][azure_cli] snippet below:
+You can find the endpoint for your Anomaly Detector resource either in the Azure Portal or by using the Azure CLI snippet below:
```

Comment:
Is there any existing doc page we can link to which goes through the steps of finding the endpoint in the portal?